docs/source/asr/configs.rst

```diff
@@ -10,7 +10,7 @@ for audio files, parameters for any augmentation being performed, as well as the
 this page cover each of these in more detail.
 
 Example configuration files for all of the NeMo ASR scripts can be found in the
-`config directory of the examples <https://github.com/NVIDIA/NeMo/tree/v1.0.2/examples/asr/conf>`_.
+`config directory of the examples <https://github.com/NVIDIA/NeMo/tree/main/examples/asr/conf>`_.
```

Comment:
Shouldn't this be stable? @titu1994

Reply:
good catch! It should be stable. will fix.